src/app/gallery/page.tsx

```diff
@@ -1,5 +1,95 @@
-export default function page() {
+"use client";
+import Image from "next/image";
+import Pagination from "../../../components/pagination/Pagination";
+import ArtworkCard from "../../../components/ArtworkCard";
+import gallery_image from "../../../public/gallery/gallery_image.svg";
+// eslint-disable-next-line
+import { useSearchParams } from "next/navigation";
+import { artworks } from "../../../mock/artworks.js";
+
+
+// the disable lint rule is because lint not interpreting useSearchParams as a react hook -> nextJS client component hook
+
+export default  function page() {
+  // eslint-disable-next-line
+  const searchParams = useSearchParams();
+  const queryPage = searchParams.get("page");
+  const page = queryPage ? parseInt(queryPage) : 1;
+  
+  const artworksPerPage = 16;
+  const startIndex = (page - 1) * artworksPerPage;
+  const endIndex = startIndex + artworksPerPage;
+  
+  const total = 30;
+  const pageData = artworks.slice(startIndex, endIndex);
+  
+
   return (
-    <div>gallery</div>
+    <div>
+      {/* yellow container */}
+      <div className="flex justify-between items-center md:flex-col py-10 px-5 bg-main-yellow">
+        <div className="text-left md:text-center">
+          <h1 className="font-bold text-4xl">Gallery</h1>
+          <p className="font-medium md:font-semi-bold text-lg">You can vote only once, so share with friends.</p>
+          <p className="font-medium md:font-semi-bold text-lg">Vote until <span className="font-extrabold"> June 29, 2014 </span> (12:00) midnight EST </p>
+        </div>
+        {/* image container */}
+        <div className="text-left md:py-5">
+          <Image src={gallery_image} alt="Previous artwork submission" />
```

Comment:
I'm not sure what this alt description means...could you make this a bit more clear?